tensorflow/workspace.bzl

```diff
@@ -390,6 +390,24 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
       build_file = str(Label("//third_party:codegen.BUILD")),
   )
 
+  native.new_http_archive(
+      name = "opencl_headers",
+      urls = [
+          "https://github.com/KhronosGroup/OpenCL-Headers/archive/f039db6764d52388658ef15c30b2237bbda49803.tar.gz",
+      ],
+      strip_prefix = "OpenCL-Headers-f039db6764d52388658ef15c30b2237bbda49803",
+      build_file = str(Label("//third_party:opencl-headers.BUILD")),
+  )
+
+  native.new_http_archive(
+      name = "opencl",
```

Comment:
nitpick: opencl -> opencl-icd?

Reply:
I can change it to opencl-icd-loader if you want to be totally accurate, sure :)